migration/migration.go

```diff
@@ -554,14 +555,12 @@ func BootstrapWorkItemLinking(ctx context.Context, linkCatRepo *link.GormWorkIte
 
 func createOrUpdateWorkItemLinkCategory(ctx context.Context, linkCatRepo *link.GormWorkItemLinkCategoryRepository, linkCat *link.WorkItemLinkCategory) (*link.WorkItemLinkCategory, error) {
 	cat, err := linkCatRepo.Load(ctx, linkCat.ID)
-	cause := errs.Cause(err)
-	switch cause.(type) {
-	case errors.NotFoundError:
+	if ok, _ := errors.IsNotFoundError(err); ok {
 		cat, err = linkCatRepo.Create(ctx, linkCat)
 		if err != nil {
 			return nil, errs.WithStack(err)
 		}
-	case nil:
+	} else {
```

Comment:
I understand why you use else but this also omits the internal server error from the load operation